deltachat-ffi/deltachat.h

```diff
@@ -318,6 +318,10 @@ char*           dc_get_blobdir               (const dc_context_t* context);
  *                    The library uses the `media_quality` setting to use different defaults
  *                    for recoding images sent with type DC_MSG_IMAGE.
  *                    If needed, recoding other file types is up to the UI.
+ * - `basic_web_rtc_instance` = address and token to webrtc signaling server (https://github.com/cracker0dks/basicwebrtc)
+ *                    that should be used for calls
```

Comment:
this should mention that the core itself does nothing with this URL, and it's solely the client side that needs to talk the "basicwebrtc" protocol (whatever it is precisely)

Reply:
and i would add an example :)

Reply:
the main reason to get it in here, however, is that it can be preset by the provider-db, so we can eg. have empty as default, but define things for testrun.org. also, i think sooner or later we will have the message to be sent out from the core.